pages/k8s/custom-loadbalancer.md

```diff
@@ -0,0 +1,61 @@
+---
+wrapper_template: "base_docs.html"
+markdown_includes:
+  nav: "shared/_side-navigation.md"
+context:
+  title: "Custom Load Balancers"
+  description: How to configure your Kubernetes cluster to use a custom load balancer.
+keywords: high availability, vip, load balancer, f5
+tags: [operating]
+sidebar: k8smain-sidebar
+permalink: custom-lb.html
+layout: [base, ubuntu-com]
+toc: False
+---
+
+The **Charmed Distribution of Kubernetes<sup>&reg;</sup>** supports setting a
+custom IP address for the control plane.  There are two ways of achieving this, depending
+on which type of loadbalancing solution you wish to configure:
+
+ -  If you have a virtual IP to place in front of machines, configure the settings on the
+    `kubeapi-load-balancer` charm.
+
+ -  If a full load balancing solution is in place such as an F5 appliance, remove the
+     `kubeapi-load-balancer` and use the settings on the `kubernetes-master` charm to
+      configure the load balancer.
+
+Both solutions are described in the sections below.
+
+# Virtual IP in front of kubeapi-load-balancer
+
+If you have a custom setup with a virtual IP in front of the kubeapi-load-balancer
```

Comment:
do we need "custom setup" here? We don't want to define how you do it, but as long as you have a virtual IP in front of the load balancers you are good.

Reply:
Ok, I will reword this.